### PR TITLE
Review for Portuguese translation of `maintenance_contributions.pt.Rmd`

### PR DESCRIPTION
**Pull request by pedropark99**
This PR brings three change to the automatic output generated by the translation tools:

- Fix small typos in some sentences;
- Fix the value of `repo-actios` YAML option, which was translated from `true` to `yes`

### CODE REVIEW

**Comment by pedropark99** [comment]
@danielvartan @marianambc @joaojcorreia @msperlin  This PR is ready for Review 2 in case any of you want to take a look at it 😉.

**Comment by danielvartan** [comment]
I can do it, but I don't believe @marianambc and @joaojcorreia have had the opportunity to contribute yet. I think it's best to wait for their responses.

File: maintenance_contributing.pt.Rmd [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
 
 # Guia de contribuição {#contributingguide}
 
 ```{block, type="summaryblock"}
-Este capítulo descreve nosso Guia de Contribuição, que descreve como você pode fazer contribuições com código e sem código para o projeto rOpenSci.
+Este capítulo apresenta o nosso Guia de Contribuição, que descreve como você pode fazer contribuições com código e sem código para a rOpenSci.
 
 ```
+-
+Então você quer contribuir para a rOpenSci? Fantástico! Nós desenvolvemos o [Guia de contribuição da comunidade rOpenSci](https://contributing.ropensci.org/) para dar as boas-vindas a você na rOpenSci e ajudá-lo a se reconhecer como um colaborador em potencial. O guia ajudará você a descobrir o que você pode ganhar doando seu tempo, conhecimento e experiência, combinando suas necessidades com coisas que ajudarão a missão da rOpenSci, e conectando você a recursos que podem te ajudar ao longo do caminho.
```
Comment (beatrizmilz) [inline maintenance_contributing.pt.Rmd:14]:
```suggestion
Então você quer contribuir para a rOpenSci? Fantástico! Nós desenvolvemos o [Guia de contribuição da comunidade rOpenSci](https://contributing.ropensci.org/) para dar as boas-vindas a você na rOpenSci e te ajudar a se reconhecer como um(a) colaborador(a) em potencial. O guia ajudará você a descobrir o que você pode ganhar doando seu tempo, conhecimento e experiência, combinando suas necessidades com coisas que ajudarão a missão da rOpenSci, e conectando você a recursos que podem te ajudar ao longo do caminho.
```

File: maintenance_contributing.pt.Rmd [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
 
-Então você quer contribuir com o rOpenSci? Você quer contribuir com o rOpenSci? Fantástico! Nós desenvolvemos o [Guia de contribuição da comunidade rOpenSci](https://contributing.ropensci.org/) para dar as boas-vindas a você na rOpenSci e ajudá-lo a se reconhecer como um colaborador em potencial. Ele o ajudará a descobrir o que você pode ganhar doando seu tempo, conhecimento e experiência, combinando suas necessidades com coisas que ajudarão a missão da rOpenSci e conectando-o a recursos para ajudá-lo ao longo do caminho.
+Nossa equipe e comunidade promovem ativamente um ambiente acolhedor em que usuários e desenvolvedores de diferentes origens e níveis de habilidade aprendem, compartilham ideias e inovam juntos abertamente por meio de normas e software compartilhados. A participação em todas as atividades da rOpenSci é apoiada por [nosso Código de Conduta](https://ropensci.org/code-of-conduct).
```
Comment (beatrizmilz) [inline maintenance_contributing.pt.Rmd:16]:
```suggestion
Nossa equipe e comunidade promovem ativamente um ambiente acolhedor em que pessoas usuárias e desenvolvedoras de diferentes origens e níveis de habilidade aprendem, compartilham ideias e inovam juntas abertamente por meio de normas e software compartilhados. A participação em todas as atividades da rOpenSci é apoiada por [nosso Código de Conduta](https://ropensci.org/code-of-conduct).
```

File: maintenance_contributing.pt.Rmd [hunk 3]
```diff
@@ -1,21 +1,21 @@
...
-
-Aceitamos contribuições com código e sem código de programadores novos e experientes em qualquer estágio da carreira e em qualquer setor. Você não precisa ser um desenvolvedor! Talvez você queira **passar 30 minutos** compartilhando o caso de uso do seu pacote em nosso fórum público ou relatando um bug, **uma hora** você pode aprender participando de uma chamada da comunidade, **cinco horas** revisando um pacote R enviado para revisão aberta por pares, **ou talvez você queira assumir um compromisso contínuo** para ajudar a manter um pacote.
+Aceitamos contribuições com código e sem código de programadores novos e experientes em qualquer estágio da carreira e em qualquer setor. Você não precisa ser um desenvolvedor! Talvez você queira **passar 30 minutos** compartilhando o caso de uso do seu pacote em nosso fórum público ou relatando um bug, **uma hora** você pode aprender participando de uma chamada da comunidade, **cinco horas** revisando um pacote de R enviado para revisão aberta por pares, **ou talvez você queira assumir um compromisso contínuo** para ajudar a manter um pacote.
```
Comment (beatrizmilz) [inline maintenance_contributing.pt.Rmd:18]:
```suggestion
Aceitamos contribuições com código e sem código de pessoas programadoras novas ou experientes em qualquer estágio da carreira e em qualquer setor. Você não precisa ser um(a) desenvolvedor(a)! Talvez você queira **passar 30 minutos** compartilhando o caso de uso do seu pacote em nosso fórum público ou relatando um bug, **uma hora** aprendendo e participando de uma chamada da comunidade, **cinco horas** revisando um pacote de R enviado para revisão aberta por pares, **ou talvez você queira assumir um compromisso contínuo** para ajudar a manter um pacote.
```

File: maintenance_contributing.pt.Rmd [hunk 4]
```diff
@@ -24,11 +24,11 @@ Quais são alguns dos benefícios de contribuir?
 - Faça e responda a novas perguntas de pesquisa conhecendo novas ferramentas de software e aliados
 - Sentir-se confiante e apoiado em seus esforços para escrever código e desenvolver software
```
Comment (beatrizmilz) [inline maintenance_contributing.pt.Rmd:25]:
```suggestion
- Sentir-se confiante e com apoio em seus esforços para escrever código e desenvolver software
```

**Review by beatrizmilz — COMMENTED** [review]
Oi Pedro!
Seguem algumas sugestões.
A principal questão é o nosso idioma ser muito marcado por gênero.
Tentei não deixar tão marcado no masculino.
Qual sua opinião? Como estão fazendo no projeto?
Se vocês estão deixando no masculino, pode ignorar as sugestões que são relacionadas à essas mudanças!

**Comment by yabellini** [comment]
Hi @pedropark99, checking this PR, I think it will be ready to merge if you resolve @beatrizmilz's suggestion.  Let me know!

File: maintenance_contributing.pt.Rmd (hunk 1 above)
Comment (pedropark99) [reply maintenance_contributing.pt.Rmd:14]: Good suggestion.

**Comment by pedropark99** [comment]
@beatrizmilz  my opinion is that I liked your suggestions, so I added them 😉. The reality is that is really hard to write gender-neutral portuguese, so, I will accept any alternative that will move our text closer to this objective. Peace ✌

@yabellini  I also understand this PR is now ready to merge.

**Pull request merged**